src/game/AI/ScriptDevAI/scripts/kalimdor/dire_maul/dire_maul.cpp

```diff
@@ -118,20 +122,48 @@ struct npc_warlock_mount_ritualAI : public  Scripted_NoMovementAI
     void JustSummoned(Creature* pSummoned) override
     {
         m_luiSummonedMobGUIDs.push_back(pSummoned->GetObjectGuid());
+        // The spawning of the waves is handled in the DB relay scripts, the movement to the center I'm doing here because simply using a 
```

Comment:
Generally all waves should be spawn groups with their defined points and each wave be a new one instead of doing this. However, I am not hellbent on someone reworking that works and doesnt clash with any overarching architecture.